js/modules/loki_public_chat_api.js

```diff
@@ -518,9 +525,12 @@ class LokiPublicChannelAPI {
   }
 
   async pollOnceForMessages() {
+    // mark sure send() can access this object instance
+    const ref = this;
```

Comment:
This looks like a binding issue?
Where is that function `send()` that needs to access this instance?

Reply:
remanent from a previous version, I think we can remove these 2 lines